apps/omg_child_chain/test/omg_child_chain/integration/happy_path_test.exs

```diff
@@ -244,6 +248,13 @@ defmodule OMG.ChildChain.Integration.HappyPathTest do
     assert {:error, %{"code" => "submit:utxo_not_found"}} = tx |> Transaction.Signed.encode() |> submit_transaction()
   end
 
+  @tag fixtures: [:alice, :in_beam_child_chain]
+  test "check submitted fee transaction is rejected", %{alice: alice} do
+    fee_tx = OMG.TestHelper.create_encoded_fee_tx(1000, alice.addr, @eth, 1000)
+
+    assert {:error, %{"code" => "submit:transaction_not_supported"}} = submit_transaction(fee_tx)
+  end
+
```

Comment:
can I assume Watcher integration tests with fee-tx-rich blocks are pending a separate issue?

Reply:
Yes, Watcher integration is the next task